openmdao/utils/gui_testing_utils.py

```diff
@@ -0,0 +1,104 @@
+"""Define utils for use in testing GUIs with Playwright."""
+import unittest
+
+
+class gui_test_case(unittest.TestCase):
```

Comment:
Maybe we should "CamelCase" this class name (i.e. GUITestCase) to conform with PEP8:
https://www.python.org/dev/peps/pep-0008/#class-names

Reply:
Changed. Thanks.